includes/admin/class-geodir-admin-dummy-data.php

```diff
@@ -752,6 +762,13 @@ public static function dummy_data_types( $post_type = 'gd_place' ) {
 			)
 		);
 
+		if( $post_type == 'gd_place' ){
```

Comment:
its wrong to limit this here, a CPT can be added by the user and can be called anything